test/test_s3_aggregator.py

```diff
@@ -46,8 +46,8 @@ def get_config(self, num_browsers=1, data_dir=""):
         return manager_params, browser_params
 
     @pytest.mark.skipif(
-        "TRAVIS" in os.environ and os.environ["TRAVIS"] == "true",
-        reason="Localstack fails to start on Travis",
+        "CI" in os.environ and os.environ["CI"] == "true",
+        reason="Localstack fails to start on CI",
```

Comment:
There are two more references to `"TRAVIS"` that we should update in [`test_extension.py`](https://github.com/vringar/OpenWPM/blob/actions/test/test_extension.py#L348L351) and [`test_mp_logger.py`](https://github.com/vringar/OpenWPM/blob/actions/test/test_mp_logger.py#L141L144)

Reply:
I changed those references

Reply:
Thanks for catching this.